jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_VRFY

```diff
@@ -29,13 +29,21 @@ YMD=${PDY} HH=${cyc} generate_com -rx COM_ATMOS_ANALYSIS
 export PYTHONPATH=${HOMEgfs}/sorc/gdas.cd/ush/:${HOMEgfs}/sorc/gdas.cd/ush/eva:${PYTHONPATH}
 export PYTHONPATH=${HOMEgfs}/sorc/gdas.cd/ush/soca:${PYTHONPATH}
 
-###############################################################
-# Run relevant script
+if [[ ${DO_VRFYOCNDA} == "YES" ]]; then
```

Comment:
why not remove this task from the workflow?
Doing it this way, it will still be generated in the workflow and then immediately exit.